app/views/layouts/application.html.erb

```diff
@@ -10,26 +10,40 @@
 <body>
 
 <!-- Navigation bar. -->
-<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+<nav class="navbar navbar-inverse khan-nav">
   <div class="container">
-    <div>
-      <%= image_tag '/favicon.ico', :class => 'img-responsive img-logo' %>
-      <%= link_to 'KhanAcademy Myanmar', root_path, :class => 'navbar-brand pull-left inline-brand' %>
+    <div class="navbar-header">
+      <%= link_to(root_path) do %>
+        <%= image_tag('/favicon.png', :class => 'img-logo') %>
+        <div class="navbar-brand"><b>Khan</b>Academy <b><i>Myanmar</i></b></div>
+      <% end %>
     </div>
-    <div class="collapse navbar-collapse">
-      <% if user_signed_in? %>
+    <% if user_signed_in? %>
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#collapse-to-nav-mini">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="collapse-to-nav-mini">
         <ul class="nav navbar-nav navbar-right">
+          <li class="nav-mini" role="presentation"><a href="#">Profile</a></li>
+          <li class="nav-mini" role="presentation"><a href="#">Settings</a></li>
+          <li class="nav-mini" role="presentation"><%= link_to('Log Out', destroy_user_session_path, :method => :delete) %></li>
           <li class="dropdown">
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown"><%= current_user.email %> <span class="caret"></span></a>
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"><%= current_user.email %> <span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
-              <li><%= link_to('Logout', destroy_user_session_path, :method => :delete) %></li>
+              <li role="presentation"><a href="#">Profile</a></li>
+              <li role="presentation"><a href="#">Settings</a></li>
+              <li class="divider"></li>
+              <li role="presentation"><%= link_to('Log Out', destroy_user_session_path, :method => :delete) %></li>
             </ul>
           </li>
         </ul>
-      <% else %>
-        <a href="<%= url_for(new_user_session_path) %>" class="btn btn-default navbar-btn navbar-right">Sign In</a>
-      <% end %>
-    </div>
+      </div>
+    <% else %>
+      <a href="<%= url_for(new_user_session_path) %>" class="btn btn-default navbar-btn navbar-signin">Sign In</a>
```

Comment:
Since we've switched to nice Rails helpers, should probably update this ugly link as well:
```
<%= link_to 'Sign In', new_user_session_path, :class => 'btn btn-default navbar-btn navbar-signin' %>
```